reference/api/aut/index.md

```diff
@@ -10,9 +10,15 @@ Interface for interacting with Autonity Contract functions using:
 - The `aut` command-line RPC client to submit calls to inspect state and state-affecting transactions.
 - JSON-RPC methods to submit calls to inspect state.
 
-{{pageinfo}}
+::: {.callout-note title="Info" collapse="false"}
 Examples for calling functions from `aut` use the setup described in the How to [Submit a transaction from Autonity Utility Tool (aut)](/account-holders/submit-trans-aut/).
-{{/pageinfo}}
+:::
+
+::: {.callout-note title="Protocol contract calls" collapse="false"}
+Some Usage and Examples illustrate using the Autonity Protocol Contract's generated ABI and the `aut` tool's `contract` command to submit transactions to the Autonity Protocol Contract address `0xBd770416a3345F91E4B34576cb804a576fa48EB1`. See `aut contract tx --help`.
```

Comment:
I think this note is too specific wrt interacting with the Autonity.sol contract.

The mechanism of passing the `.abi` file to aut can be used to interact with any protocol contract (or more generally with every contract, but we don't care much about it here).

Anyways I would suggest to make this note general (i.e. ABIs can be found in this dir and passed to `aut in this way), then maybe use the autonity contract only as an example.

Reply:
Hmm. It's also legacy since `aut validator register` is fixed in `aut` dev 0.4 release.

Removed.

The issue https://github.com/autonity/docs.autonity.org/issues/111 has been updated to include where to find ABIs, `aut contract call`, `aut contract tx`, `aut token.`